lib/git-manager.js

```diff
@@ -410,17 +409,13 @@ function getCommit(repo, sha) {
    });
 }
 
-function getCommitStatus(repo, sha) {
+function getCommitStatuses(repo, sha) {
    return new Promise(function(resolve, reject) {
```

Comment:
It seems kind of weird that we're wrapping a promise in our own promise instead of just passing the `api.getCommitStatuses` promise.

Reply:
I agree. We're just passing it through here and losing any rejections.